0-bootstrap/README-Jenkins.md

```diff
@@ -142,14 +163,14 @@ You arrived to these instructions because you are using the `jenkins_bootstrap`
 
 1. Run terraform commands.
     - After the credentials are configured, we will create the `cft-seed` project (which contains the GCS state bucket and Terraform custom service account) and the `prj-cicd` project (which contains the Jenkins Agent, its custom service account and where we will add VPN configuration)
-    - **Use Terraform 0.12.24** to run the terraform script with the commands below:
     - **WARNING: Make sure you have commented-out the `cloudbuild_bootstrap` module and enabled the `jenkins_bootstrap` module in the `./main.tf` file**
+    - **Use Terraform 0.12.24** to run the terraform script with the commands below
```

Comment:
Is there a reason why we are recommending just `0.12.24`?

Reply:
This is because the errors I saw in my Jenkins about different versions.

All automated scripts use 0.12.24. If you use a newer version during 0-bootstrap, then, our scripts will use an older version to create the state in 1-org. This will create states with different versions.

Reply:
@bharathkkb - if this is ok, we can merge this. I won't send more changes in this PR

Reply:
I can do a repro to confirm, but my understanding was that as long as its a separate prefix, we can use different TF versions. 
i.e - as bootstrap is done locally it and the state is stored in `terraform/bootstrap/state` it doesnt matter what TF version is used to manage state in other prefixes `terraform/org/state`, `terraform/networks/state` etc
Not a blocker though.

Reply:
you are right, the states are kept in separate directories in the GCS bucket. The problem I see is that, even though Jenkins should not be automatically deploying code or even using the bootstrap step... having the terraform state with different versions is not a good thing (we don't know how people will use the codebase and I think it is better to alert them about it than leaving them in the dark)